Dependencies for opensuse?

What dependencies would I need for opensuse tumbleweed?
I get this error while compiling 


CMake Error at CMakeLists.txt:28 (find_package):
  Could not find a package configuration file provided by "ECM" with any of
  the following names:

    ECMConfig.cmake
    ecm-config.cmake

  Add the installation prefix of "ECM" to CMAKE_PREFIX_PATH or set "ECM_DIR"
  to a directory containing one of the above files.  If "ECM" provides a
  separate development package or SDK, be sure it has been installed.

--- a/README.md
+++ b/README.md
@@ -61,7 +61,7 @@ openSUSE:
 sudo zypper in vulkan-devel plasma-framework-devel plasma5-workspace-devel \
 libqt5-qtwebsockets-devel mpv-devel python310-websockets \
 libqt5-qtx11extras-devel liblz4-devel gstreamer-plugins-libav \
-libqt5-qtbase-private-headers-devel cmake
+libqt5-qtbase-private-headers-devel cmake extra-cmake-modules
 ```
 
 Fedora Kinoite:  

